scripts/scenarios/generate.py

```diff
@@ -144,18 +144,31 @@ def main(scenarios: list[Path], snapshot_update: bool = True):
             else []
         )
 
-    # TEMPORARY
-    # We do not yet support local version identifiers
+    # We don't yet support local versions that aren't expressed as direct dependencies.
     for scenario in data["scenarios"]:
         expected = scenario["expected"]
-        if (
-            scenario["name"].startswith("local-")
-            and scenario["name"] != "local-not-latest"
+        if scenario["name"] in (
+            "local-less-than-or-equal",
+            "local-simple",
+            "local-transitive-confounding",
+            "local-used-without-sdist",
         ):
             expected["satisfiable"] = False
-            expected[
-                "explanation"
-            ] = "We do not have correct behavior for local version identifiers yet"
+            expected["explanation"] = (
+                "We do not have correct behavior for local version identifiers yet"
+            )
+        elif scenario["name"] == "local-greater-than":
+            expected["satisfiable"] = True
+            expected["packages"] = [
+                {
+                    "name": "local-greater-than-a",
```

Comment:
I think this one is fixable with tricks similar to what we do for prereleases with "min version" thing.